app/ui/src/app/customizations/api-connector/api-connector-create/api-connector-auth/api-connector-auth.component.ts

```diff
@@ -47,9 +47,15 @@ export class ApiConnectorAuthComponent implements OnInit, OnDestroy {
       .get('authenticationType')
       .valueChanges.subscribe(value => this.setOAuthFormValidation(value));
 
-    this.setOAuthFormValidation(
-      authenticationType ? authenticationType.defaultValue : ''
-    );
+    this.setOAuthFormValidation( defaultAuthenticationType );
```

Comment:
Wonder if `this.setOAuthFormValidation( defaultAuthenticationType ? defaultAuthenticationType : '' );` might be good. In reality, this code doesn't make sense (the `setOAuthFormValidation` method), not referring to your fix, considering that it gets passed a boolean, expects a string, and doesn't fail. We really should start setting return types, otherwise defeats the purpose of using TS.